#! /bin/sh /usr/share/dpatch/dpatch-run
## 18_gpsd.hotplug_location.dpatch by Tilman Koschnick <til@subnetz.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adjust WHEREAMI variable in gpsd.hotplug.

@DPATCH@
--- a/gpsd.hotplug
+++ b/gpsd.hotplug
@@ -7,7 +7,7 @@
 import sys, time, os, syslog, glob, socket, stat
 
 CONTROL_SOCKET = "/var/run/gpsd.sock"
-WHEREAMI = "/etc/hotplug/usb/gpsd.hotplug"
+WHEREAMI = "/lib/udev/gpsd.hotplug"
 
 def gpsd_control_connect():
     "Acquire a connection to the GPSD control socket."
